### ISSUE TITLE
Selectors 4: test suite and implementation report

### ISSUE THREAD

**Issue by svgeesus** [w3c/csswg-drafts#10052]
The [/TR of Selectors 4](https://www.w3.org/TR/selectors-4/) is currently a [WD published 11 Nov 2022](https://www.w3.org/TR/2022/WD-selectors-4-20221111/). It has the old test results widget, which proudly declares **133** tests.

![image](https://github.com/w3c/csswg-drafts/assets/2506926/7d96ed41-0f75-4123-bfcb-ef8256ba446f)

That link is to the old, non-maintained, manual test suite.

Looking now at WPT, there are [**5436** tests](https://wpt.fyi/results/css/selectors?label=experimental&label=master&aligned) across all levels, which are well maintained and run every day.

![image](https://github.com/w3c/csswg-drafts/assets/2506926/3fa15a00-44d5-47f2-8c24-8ec1c20d3086)

Thus, it would be valuable to add the [Bikeshed WPT annotations](https://speced.github.io/bikeshed/#testing) to that spec, indicating which spec sections are tested by which tests. Clearly that is a lot of work, but it would make the spec significantly more useful and help with advancement to CR.

**Comment by svgeesus** [comment]
@fantasai @tabatkins

**Comment by tabatkins** [comment]
Yes, we saw this issue, but as you said, it'll be a big job to fill all those in. Wasn't much use chiming in with a +1 post.
